workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/GameObjectCreation/GameObjectInitializationSystem.cs

```diff
@@ -38,8 +32,8 @@ private struct RemovedEntitiesData
 
         private Worker worker;
         private ViewCommandBuffer viewCommandBuffer;
-        private EntityGameObjectCreator entityGameObjectCreator;
         private EntityGameObjectLinker entityGameObjectLinker;
+        private EntityManager entityManager;
         private readonly Dictionary<int, GameObject> entityGameObjectCache = new Dictionary<int, GameObject>();
```

Comment:
gonna merge in a fix for a bug in this system in the next half hour. Make sure to rebase on top of that (it will probably create some merge conflicts anyway)